lib/rouge/lexers/puppet.rb

```diff
@@ -64,6 +64,13 @@ def self.metaparameters
         rule /{/, Punctuation, :regex_allowed
         rule /,/, Punctuation, :regex_allowed
 
+        # Handle the = operator.
+
+        # The backslash is required here to prevent Ruby from
+        # interpreting /= as the divide-and-assign operator!
+        #
+        rule /\=/, Operator
+
```

Comment:
Would it make sense to add `=` to the `rule %r([<>!+*/-]), Operator` regex below? This would also likely have the added benefit of not needing the leading slash and the comment wouldn't be needed